USER_MANUAL.md

```diff
@@ -1290,11 +1290,11 @@ Method costs. Currency is not important but must be consistent across all inputs
 
 **Default input:** N/A
 
-**Description:** Allows for surveying methods to determine which follow-up method to run if multiples are present. If not set, LDAR-Sim will expect only 1 followup method to be present for a single program.
+**Description:** Allows for surveying methods to determine which follow-up method to run if multiples are present. If not set, LDAR-Sim will expect only 1 followup method to be present for a single program. Site level follow-up methods can be used by leveraging this parameter, allowing for a subsequent rounds of screenings based on initial screening results (In order to increase confidence in fugitives, measurement accuracy, etc). The site level follow-up must be set as the preferred_method for the initial screening.
```

Comment:
Please remove the 'a' between "allowing for a subsequent..." 
I would also remove the brackets around "In order to..." and just have it as a sentence. Maybe reword it by saying "This feature can be used to increase confidence... " 
Suggestion in rewording the last sentence: Each method in the follow-up chain must have the preferred_method set to the corresponding method it triggers.